src/Sql/dbo/Tables/Cache.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE TABLE [dbo].[Cache] (
+    [Id] [nvarchar](449) NOT NULL,
+    [Value] [varbinary](max) NOT NULL,
+    [ExpiresAtTime] [datetimeoffset](7) NOT NULL,
+    [SlidingExpirationInSeconds] [bigint] NULL,
+    [AbsoluteExpiration] [datetimeoffset](7) NULL,
```

Comment:
⛏️ Minor casing and syntax differences from what we normally do.

Reply:
Yes, this just matches the generated table expected for SqlServer IDistCache when you run the `dotnet sql-cache create` command. See https://learn.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-8.0#distributed-sql-server-cache